rllib/evaluation/collectors/simple_list_collector.py

```diff
@@ -179,11 +181,20 @@ def build(self, view_requirements: ViewRequirementsDict) -> SampleBatch:
                                           view_req.shift_to + 1 + obs_shift]
                         for i in range(count)
                     ])
+                # Batch repeat value = 1: Repeat the shift_from/to range at
+                # each timestep.
                 else:
-                    data = np_data[data_col][self.shift_before +
-                                             view_req.shift_from +
-                                             obs_shift:self.shift_before +
-                                             view_req.shift_to + 1 + obs_shift]
+                    d = np_data[data_col]
```

Comment:
Why not the code before? It is more concise.

Reply:
We have to use the `np.lib.stride_tricks.as_strided` to achieve the "frame" stacking. Didn't want to do all these `np_data[data_col]` lookups, so I shortened to `d`.